From: Tatyana Nikolova <Tatyana.E.Nikolova@intel.com>
Subject: [PATCH] RDMA/nes: TSO is enabled again for linux-3.5 - Backport change

RDMA/nes: TSO is enabled again for linux-3.5
(the patch should be applied on top of previous backports)

Signed-off-by: Tatyana Nikolova <Tatyana.E.Nikolova@intel.com>
---
 drivers/infiniband/hw/nes/nes_nic.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/drivers/infiniband/hw/nes/nes_nic.c b/drivers/infiniband/hw/nes/nes_nic.c
index xxxxxxx..xxxxxxx xxxxxx
--- a/drivers/infiniband/hw/nes/nes_nic.c
+++ b/drivers/infiniband/hw/nes/nes_nic.c
@@ -1760,12 +1760,10 @@ struct net_device *nes_netdev_init(struct nes_device *nesdev,
 	netdev->features |= NETIF_F_HIGHDMA | NETIF_F_HW_VLAN_TX;
 
 	if ((nesvnic->logical_port < 2) || (nesdev->nesadapter->hw_rev != NE020_REV)) {
-#if (LINUX_VERSION_CODE <= KERNEL_VERSION(3,3,0))
 		netdev->features |= NETIF_F_TSO;
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(2,6,39))
 		netdev->hw_features |= NETIF_F_TSO;
 #endif
-#endif
 	}
 
 	nes_debug(NES_DBG_INIT, "nesvnic = %p, reported features = 0x%lX, QPid = %d,"
